src/libImaging/Paste.c

```diff
@@ -425,7 +425,7 @@ fill_mask_L(
                 *out = BLEND(*mask, *out, ink[0], tmp1);
                 if (strncmp(imOut->mode, "I;16", 4) == 0) {
                     out++;
-                    *out = BLEND(*mask, *out, ink[0], tmp1);
+                    *out = BLEND(*mask, *out, ink[1], tmp1);
```

Comment:
why?

Reply:
`ink` was originally an `int` in `_draw_bitmap`
https://github.com/python-pillow/Pillow/blob/7a1e28404d692d4a7fed33dc67019a1d8c4bf9d9/src/_imaging.c#L2898
and then became `const void *ink` for `ImagingDrawBitmap`.
https://github.com/python-pillow/Pillow/blob/7a1e28404d692d4a7fed33dc67019a1d8c4bf9d9/src/libImaging/Draw.c#L834
It passed through `ImagingFill2`, and then became `const UINT8 *ink` for `fill_mask_L`
https://github.com/python-pillow/Pillow/blob/7a1e28404d692d4a7fed33dc67019a1d8c4bf9d9/src/libImaging/Paste.c#L403

The second item of that `UINT8` pointer array will give us the second part of the 16 bits.

Reply:
@homm Any further concerns, or good to merge?